components/vf-summary/vf-summary.config.yml

```diff
@@ -31,3 +86,5 @@ variants:
     hidden: true
     context:
       size: s
+  - name: event-easy
```

Comment:
Is this intended to be a new variant?

Reply:
It was, an idea from the before times that never got into being used. So I’ve hidden it here. 

https://latest.visual-framework.dev//components/vf-summary/index.html?q=Summary#vf-summary--event-easy

Reply:
ah, didn't realise that was live. Yeah, the event container. Was a nice design from Mark, but would probably need an overhaul given updated context.